TESTING.md

```diff
@@ -50,7 +50,7 @@ To run specific functional tests, you can run by CI group:
 
 To debug functional tests:
 Say that you would want to debug a test in CI group 1, you can run the following command in your environment:
-`node --debug-brk --inspect scripts/functional_tests.js --config test/functional/config.js --include ciGroup1 --debug`
```

Comment:
Does this work!? I know it starts for me but it doesn't really help.

Reply:
yep. thats why I changed the flag name